docs/tutorials/typescript-deploy.md

```diff
@@ -0,0 +1,206 @@
+# Deploy you smart contract to the Koinos blockchain
+
+Originally published by Roamin on [Hashnode](https://hashnode.com/@Roamin)
+
+Today we're going to learn how to deploy a smart contract to the Koinos blockchain. We won't be writing any code, so if you don't have a compiled Koinos smart contract ready to deploy, I invite you to check my previous blog where we learned how to "[Create your first Koinos smart contract in minutes](/first-smart-contract-on-koinos.md)".
```

Comment:
The documentation site shouldn't be phrased in 1st person. We should remove/modify this.